hack/dev-env.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+RDR_NAME_PREFIX=${RDR_NAME_PREFIX:-rdr}
+script_dir="$(cd "$(dirname "$0")" && pwd)"
+old_cwd=$(pwd)
+
+function finish {
+    cd "$old_cwd"
+}
+trap finish EXIT
+
+if [[ $1 != "create" && $1 != "destroy" ]]; then
+    echo "Usage: $0 create|destroy"
+    exit 1
+fi
+
+cd "$script_dir"/..
+make venv > /dev/null
+# shellcheck disable=SC1091
+source venv > /dev/null
```

Comment:
Unfortunately we cannot hide this from the user, since ramenctl and basic-test depends on the virtual env, so it will be best to do this manually.

Also the virtual env is a good place to deploy tools automatically so developers do not need to deploy stuff manually, and tools deployed n the virtual env do not affect tools deployed on the host.

Reply:
Yes and I am not removing virtual env, just abstracting it. With this change, we have abstracted the use of virtual env in the case of drenv. We can also do the same for ramenctl and basic test.

Later on, we can rewrite ramenctl and basic test in Go.

Reply:
I updated the code to not hide the virtual environment activation now.